packages/components/src/molecules/Carousel/CarouselItem.tsx

```diff
@@ -18,19 +19,24 @@ function CarouselItem({
   totalItems,
   infiniteMode,
   isScrollCarousel,
+  marginRightValue,
 }: PropsWithChildren<CarouselItemProps>) {
   const { isItemVisible, shouldRenderItem } = useSlideVisibility({
     totalItems,
     currentSlide: state.currentItem,
     itemsPerPage: state.itemsPerPage,
   })
 
-  const style =
-    ((!isScrollCarousel && { width: '100%' }) as CSSProperties) ||
-    ((isScrollCarousel && {
-      maxWidth: '60%',
-      display: 'inline-block',
-    }) as CSSProperties)
+  const defaultStyle = { width: '100%' } as CSSProperties
+  const scrollCarouselStyle = {
+    width: `calc((100% - ${marginRightValue} * ${state.itemsPerPage - 1}) / ${
```

Comment:
@hellofanny  this change is breaking our layout 
![image](https://github.com/vtex/faststore/assets/137450920/62244702-419f-4fa8-9305-9876c4e89254)

when the user resize the screen

Reply:
Hi Victor! 😊 As mentioned [here](https://www.faststore.dev/components/molecules/carousel#using-productcard-component) this behavior is expected, after discussing with the design team, take this approach to safeguard the content of the ProductCard component. I'm not sure about your expected layout, perhaps consider displaying fewer itemsPerPage specifically for this breakpoint.